make_beam/get_delays_small.c

```diff
@@ -428,16 +457,31 @@ void get_delays(
                             for (p1 = 0; p1 < NPOL;  p1++)
                             for (p2 = 0; p2 < NPOL;  p2++)
                                 invJi[ant][ch][p1][p2] = CMaked( 0.0, 0.0 );
+                            }
                         }
                     }
-                }
 
             } // end loop through antenna/pol (row)
         } // end loop through fine channels (ch)
-
+        /*
+        if (beam_model == BEAM_ANALYTIC)
+        {
+            for (ch = 0; ch < NCHAN; ch++)
+            {
+                for (p1 = 0; p1 < NPOL;  p1++)
+                {
+                    for (p2 = 0; p2 < NPOL;  p2++)
+                    {
+                        fprintf(stderr, "%.18lf %.18lf ", CReald(invJi[5][ch][p1][p2]), CImagd(invJi[5][ch][p1][p2]));
+                    }
+                }
+                fprintf(stderr, "\n");
+            }
+        }
+        */
```

Comment:
Looks like this is just debugging, might want to remove it or label it as debugging